Tests/API/ApiIBTests.cs

```diff
@@ -0,0 +1,110 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Web;
+using NUnit.Framework;
+using QuantConnect.Api;
+using QuantConnect.API;
+using QuantConnect.Brokerages;
+using QuantConnect.Configuration;
+
+namespace QuantConnect.Tests.API
+{
+    [TestFixture, Ignore("These tests require a configured and active account to Interactive Brokers")]
+    class ApiIBTests
+    {
+        private int _testAccount;
+        private string _testToken;
+        private string _dataFolder;
+        private Api.Api _api;
+        private const bool stopLiveAlgos = true;
+
+        /// <summary>
+        /// Run before test
+        /// </summary>
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            _testAccount = Config.GetInt("job-user-id");
+            _testToken = Config.Get("api-access-token");
+            _dataFolder = Config.Get("data-folder");
+
+            _api = new Api.Api();
+            _api.Initialize(_testAccount, _testToken, _dataFolder);
+        }
+
+
+        /// <summary>
+        /// Live paper trading via IB.
+        /// </summary>
+        [Test]
+        public void LiveEquityAlgorithms_CanBeUsedWithInteractiveBrokers_Successfully()
+        {
+            var user = Config.Get("ib-user-name");
+            var password = Config.Get("ib-password");
+            var account = Config.Get("ib-account");
+
+            var file = new ProjectFile
+            {
+                Name = "main.cs",
+                Code = File.ReadAllText("../../../Algorithm.CSharp/BasicTemplateAlgorithm.cs")
+            };
+
+
+            // Create a new project
+            var project = _api.CreateProject($"Test project - {DateTime.Now.ToStringInvariant()}", Language.CSharp);
+
+            // Add Project Files
+            var addProjectFile = _api.AddProjectFile(project.Projects.First().ProjectId, file.Name, file.Code);
+            Assert.IsTrue(addProjectFile.Success);
+
+            // Create compile
+            var compile = _api.CreateCompile(project.Projects.First().ProjectId);
+            Assert.IsTrue(compile.Success);
+
+            // Create default algorithm settings
+            var settings = new InteractiveBrokersLiveAlgorithmSettings(user,
+                                                                       password,
+                                                                       account);
+
+            // Wait for project to compile
+            Thread.Sleep(10000);
```

Comment:
Think we could replace this (and the other cases) by a call to `_api.ReadCompile()` every 1sec, also as a way of testing that endpoint